### DISCUSSION TITLE
Secure text with last character visible

### DISCUSSION THREAD

**Discussion by dicash** [uNmAnNeR/imaskjs#1041] [Q&A]
**What problem you are trying to solve?**
At the moment we have `displayChar`, allowing to hide characters that are not supposed to be visible.
```
IMask(element, {
  mask: '000000',
  displayChar: '•',
  lazy: false,
  overwrite: 'shift',
})
```
New requirement that we have is displaying last input character as visible, like it does iOS.

**Describe the solution you'd like**
Some option to display last character, or maybe allow passing `displayChar` a function, so that we can decide which char should be visible which shouldn't.

**Describe alternatives you've considered**
Are there alternatives?

**Comment by uNmAnNeR** [comment]
hi. yes, you can change mask `displayChar` depending on the input value + set a timeout
